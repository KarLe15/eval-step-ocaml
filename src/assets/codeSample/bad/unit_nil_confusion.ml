let _ =
  if true then ()
  else []
