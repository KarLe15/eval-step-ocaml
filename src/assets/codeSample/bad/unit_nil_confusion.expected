File "", line 3, characters 7-9:
Error: This variant expression is expected to have type unit
       The constructor [] does not belong to type unit
